packages/office-ui-fabric-react/src/components/Dropdown/Dropdown.tsx

```diff
@@ -301,7 +302,7 @@ export class Dropdown extends BaseComponent<IDropdownInternalProps, IDropdownSta
           onDismiss={ this._onDismiss }
           onPositioned={ this._onPositioned }
         >
-          <div style={ { width: this._dropDown.clientWidth - 2 } }>
+          <div style={ { width: dropdownWidth || this._dropDown.clientWidth - 2 } }>
```

Comment:
I think there is a small bug here in some browsers that show scroll bars. I've noticed that in Edge if you render a dropdown with lots of options, the width is on the content and not the scroll area. I'm wondering if we're setting the width on the container, or on the content within the scrollable area.